index.js

```diff
@@ -93,13 +94,15 @@ function _javascript (entry, opts, setCss) {
     cache: {}
   }
 
-  opts = xtend(base, opts || {})
+  const jsOpts = xtend(base, opts.js || {})
 
-  const b = (this.optimize)
-    ? browserify(opts)
-    : watchify(browserify(opts))
-  b.plugin(cssExtract, { out: createCssStream })
-  b.transform(sheetify)
+  const b = opts.optimize
```

Comment:
It looks like its in scope, but then again, I was having some weird assignment issues over in https://github.com/yoshuawuyts/bankai/issues/87#issuecomment-261153648 which smelt of scope issues.